### PR TITLE
change reference cube to post pressure/temperature regularly vs waiti…

### PR DESCRIPTION
**Pull request by bmorgan21**
…ng for commands over a socket connection

### CODE REVIEW

File: README.md
Comment (snorklerjoe) [inline README.md:1]: Thanks- I guess I forgot to change that when forking from the beacon code

File: code.py [hunk 1]
```diff
@@ -1,106 +1,91 @@
...
-import reference_config
+
+@contextmanager
+def connection():
+	c = servercom.Connection()
+	try:
+		yield c
+	finally:
+		c.close()
+
```
Comment (snorklerjoe) [inline code.py:17]: [Like in the beacon code](https://github.com/Windham-High-School/CubeServer-beacon/pull/2#discussion_r1208065542), CircuitPython does not implement this.

```suggestion
connection = servercom.Connection
```

**Review by snorklerjoe — CHANGES_REQUESTED** [review]

**Review by snorklerjoe — APPROVED** [review]
Looks good- Thanks!
I'll incorporate the new method into the scoring algos later today

File: code.py (hunk 1 above)
Comment (bmorgan21) [reply code.py:18]: import fixed.

**Pull request merged**